### DISCUSSION TITLE
Networking/WalletConnectNetworking class not exposed in the V2 swift SDK?

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1572] [Developer Support]
There is a mention of using a Networking/WalletConnectNetworking class, but i'm not seeing this class exposed via the library? Any ideas? 

<img width="839" alt="Screen_Shot_2022-10-31_at_7 01 49_PM" src="https://user-images.githubusercontent.com/36456059/199093008-3931e5b7-e433-4101-a1d6-5338ea80b5c5.png">

**Comment by GrahamMcBain** [comment]
Looks like it's also an issue for the auth class 
<img width="831" alt="Screen_Shot_2022-10-31_at_7 30 20_PM" src="https://user-images.githubusercontent.com/36456059/199094030-d7d78ba2-34d1-4c05-bccc-7a59c7753627.png">

**Reply by llbartekll** [reply]
Hey, you need to import walletconnectauth and walletconnectnetworking to use them

**Reply by DipoAreoye** [reply]
Hey, are these part of the WalletConnectV2 swift  library? Cannot see WalletConnectAuth or WalletConnectNetworking

**Reply by llbartekll** [reply]
Yes, it’s the same repo

**Reply by DipoAreoye** [reply]
<img width="322" alt="Screen Shot 2022-10-31 at 10 22 54 PM" src="https://user-images.githubusercontent.com/5705062/199121402-17664e76-a28d-4450-8c67-a76b7e7e37f7.png">
This is all I see when trying to import no mention of walletconnectauth and walletconnectnetworking

**Reply by flypaper0** [reply]
@DipoAreoye try to import package again, if you're doing it with XCode Dependency manager, select the frameworks you need

Networking and Pairing frameworks is always required to select

<img width="668" alt="Screenshot 2022-11-01 at 09 15 26" src="https://user-images.githubusercontent.com/5384197/199151233-87e998cd-6ea8-4bd1-8848-e2b4d4d4444e.png">

**Reply by DipoAreoye** [reply]
Ah awesome, Thanks! That resolved it :)

**Reply by Eunno-An** [reply]
> @DipoAreoye try to import package again, if you're doing it with XCode Dependency manager, select the frameworks you need
> 
> Networking and Pairing frameworks is always required to select
> 
> <img alt="Screenshot 2022-11-01 at 09 15 26" width="668" src="https://user-images.githubusercontent.com/5384197/199151233-87e998cd-6ea8-4bd1-8848-e2b4d4d4444e.png">

I have same problem. I can't find this frameworks... 
plz somebody help me..
<img width="847" alt="image" src="https://user-images.githubusercontent.com/33450365/199659216-45e89259-3c41-472c-bda4-dd3ca3e574f2.png">
